.github/workflows/test.yml

```diff
@@ -872,6 +872,40 @@ jobs:
       matrix:
         spec:
           [
+            './portal/cypress/e2e/journeys/*.spec.js',
+            './portal/cypress/e2e/journeys/homepage/**/*.spec.js',
+            './portal/cypress/e2e/journeys/admin/**/*.spec.js',
+            './portal/cypress/e2e/journeys/checker/**/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-deals/filter-by/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-deals/filters/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-deals/sort/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-facilities/filter-by/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-facilities/filters/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-facilities/sort/*.spec.js',
+            './portal/cypress/e2e/journeys/feedback/**/*.spec.js',
+            './portal/cypress/e2e/journeys/login/**/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/check-deal-details/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/clone-deal/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/dashboard/**/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/submit-a-deal-for-review/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/submit-a-deal-for-review-with-issued-and-unissued-facilities-and-resubmit-after-checker-returns/*.spec.js',
```

Comment:
Out of interest, is there a reason to not just do e.g.
`./portal/cypress/e2e/journeys/maker/`
instead of all of the subsets like: 
```
'./portal/cypress/e2e/journeys/maker/clone-deal/*.spec.js',
'./portal/cypress/e2e/journeys/maker/dashboard/**/*.spec.js',
'./portal/cypress/e2e/journeys/maker/submit-a-deal-for-review/*.spec.js',
```
(Aware this is likely copied from the portal structure above)